src/classes/CRLP_RollupCMT.cls

```diff
@@ -297,7 +297,7 @@ public class CRLP_RollupCMT {
             this.isActive = rlp.Active__c;
 
             try {
-                this.filterGroupLabel = rlp.Filter_Group__r.MasterLabel;
+                this.filterGroupLabel = String.isBlank(rlp.Filter_Group__r.MasterLabel) ? System.Label.CRLP_NoFilterGroupSelected : rlp.Filter_Group__r.MasterLabel;
```

Comment:
This should probably be `rlp.Filter_Group__c == null ? System.Label.CRLP_NoFilterGroupSelected : rlp.Filter_Group__r.MasterLabel;` since `rlp.Filter_Group__r.MasterLabel` will likely throw an exception when `rlp.Filter_Group__c` is null.